.github/workflows/tests.yml

```diff
@@ -266,6 +266,8 @@ jobs:
 
       - name: Install Towncrier
         run: |
+          python -m venv towncrier_check
+          source towncrier_check/bin/activate
```

Comment:
Would `pip install --user towncrier` work?

Reply:
I did not try, but from what I understand in https://github.com/pypa/pip/issues/12085, it should not work.
`--break-system-packages` would work and towncrier should be safe, but environment is the official solution.